packages/datadog-instrumentations/src/child_process.js

```diff
@@ -189,7 +186,7 @@ function wrapChildProcessAsyncMethod (ChildProcess, shell = false) {
 
       const context = createContextFromChildProcessInfo(childProcessInfo)
       const abortController = new AbortController()
```

Comment:
this abortcontroller was never passed to a subscriber no ?

Reply:
> was never passed to a subscriber

This PR is not pointing to master, is pointing to `conti/remove-async-storage-child-process` branch, it is passed to a subscriber in master, but the reason why the tests are failing in conti's PR is that is not passed there. Just fixing it.